### PR TITLE
[BACKLOG-37239][BACKLOG-37233]-Focus and Keyboard issues on Browser Files

### PR DESCRIPTION
**Pull request by phalguni-a**

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 14m 11s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl user-console
```
---
:ok_hand: All tests passed!
 
Tests run: **12**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5266/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by phalguni-a** [comment]
@pentaho/wcag , Please review

File: user-console/src/main/resources/org/pentaho/mantle/public/browser/js/browser.js [hunk 1]
```diff
@@ -1164,6 +1165,10 @@ define([
       if ($target.hasClass("open")) {
         $target.children(".element").attr("aria-expanded", false);
         $target.removeClass("open").find("> .folders").hide();
+        if ($target.find("[tabindex=0]").length > 0) {
+          $target.find("[tabindex=0]").attr("tabindex",-1);
+          $target.children(".element").attr("tabindex",0);
```
Comment (dcleao) [inline user-console/src/main/resources/org/pentaho/mantle/public/browser/js/browser.js:1170]: Formatting is off.

**Review by dcleao — CHANGES_REQUESTED** [review]
All good except formatting.

**Review by hv-vamsi — APPROVED** [review]

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 13m 11s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl user-console
```
---
:ok_hand: All tests passed!
 
Tests run: **12**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5266/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 12m 38s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl user-console
```
---
:ok_hand: All tests passed!
 
Tests run: **12**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5266/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**